Gemfile

```diff
@@ -76,6 +76,7 @@ gem 'strong_parameters'
 gem 'delayed_job_active_record', '0.3.3'
 gem 'daemons'
 
+gem 'rack', '~>1.4.6'
```

Comment:
@myabc thats probably better, I will move it. Just a hint: 037c722 states to use 1.4.5 but you probably knew this already.